_episodes/05-questions.md

```diff
@@ -13,6 +13,61 @@ keypoints:
 - "A well detailed research question can be answered with a good chart." 
 ---
 
+## Gapminder Ignorance Project
```

Comment:
I think that is its name. The foundation name is Gapminder. The project name is Ignorance. I got these general questions from the about section of the Ignorance Project - https://www.gapminder.org/ignorance/.